extensions/wikia/FacebookPreferences/templates/FacebookPreferencesModuleService_disconnected.php

```diff
@@ -0,0 +1,8 @@
+<div class="facebook-preferences">
+	<div id="fbConnectPreferences">
+		<p><?= wfMessage( 'fbconnect-convert' )->escaped(); ?></p>
+		<span id="facebook-connect-button" class="wds-button facebook-button" role="button">
```

Comment:
Buttons have default styling applied by oasis skin, and it is quite cumbersome to override that with WDS